docs/docfx-template/partials/class.tmpl.partial

```diff
@@ -0,0 +1,238 @@
+{{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
```

Comment:
I assume this file is just copied from MS somewhere?
Should it still have the MS copyright?